fdbserver/VersionedBTree.actor.cpp

```diff
@@ -6175,8 +6396,8 @@ class VersionedBTree {
 		MutationBuffer::const_iterator mBegin = mutations->upper_bound(all.subtreeLowerBound.key);
 		--mBegin;
 		MutationBuffer::const_iterator mEnd = mutations->lower_bound(all.subtreeUpperBound.key);
-
-		wait(commitSubtree(self,
+		wait(commitSubtree(0,
```

Comment:
The root level is not `0`, it is `self->m_pHeader->height`.  You can see it being used to calculate the `isLeaf` argument later in the argument list.  Having both `level` and `isLeaf` is redundant of course but I've already changed this on my branch so just keep them both for this PR.